### PR TITLE
benchmark for the Macchiato ClojureScript framework

### PR DESCRIPTION
**Pull request by yogthos**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
Hey @yogthos! This is looking good.. it just needs a line in the `.travis.yml` file

**Comment by yogthos** [comment]
ah right missed that part, just added it in

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ env:
     - "TESTDIR=Clojure/compojure"
     - "TESTDIR=Clojure/http-kit"
     - "TESTDIR=Clojure/luminus"
+      "TESTDIR=Clojure/macchiato"
```
Comment (nbrady-techempower) [inline .travis.yml]: You need the `-` to precede it. See the lines above and below

**Comment by yogthos** [comment]
oops fixed

File: frameworks/Clojure/macchiato/setup.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+fw_depends postgresql nodejs
```
Comment (nbrady-techempower) [inline frameworks/Clojure/macchiato/setup.sh]: You'll need to add leiningen here to use `lein package`

**Comment by yogthos** [comment]
Right makes sense, just updated.

**Comment by yogthos** [comment]
@nbrady-techempower looks like everything's good to go :)

**Comment by nbrady-techempower** [comment]
@yogthos Yes, I've added the proper label to it. Round 14 final is currently running, so this won't make it in for that. Am going to merge right after that run is done. Thanks!

**Comment by yogthos** [comment]
Thanks, sounds great. Since it won't make it in for the current round, I might play around with it to see if I can improve performance and update the PR if I get any improvements.

**Pull request merged**